### DISCUSSION TITLE
Question about N&N criterion

### DISCUSSION THREAD

**Discussion by elsazac** [eclipse-platform/eclipse.platform.ui#1589] [General]
I have a handful of PRs that made UI changes landed recently. Just wondering : 
 - How do we qualify a feature for N&N ?
 - Who qualifies it ?
 - Who creates N&N PRs for the designated ones ?

**Comment by merks** [comment]
If it's interesting for the framework user or the end user to know about the new feature in order to use the new feature or to avoid workarounds for the lack of the feature in the past then no doubt it's worth an N&N.  I don't think we've ever suffered from too many N&N entries so I'm doubtful there will be objections to well-written entries. 

Typically the person who implemented the feature writes the N&N, and sometimes, given available time, I review then and improve them.   E.g., this one is in my queue for tweaking:

https://github.com/eclipse-platform/www.eclipse.org-eclipse/commit/76dc491da76762fd403c4cf44c24bcfb7e93865e

**Reply by elsazac** [reply]
Thanks @merks
